src/messaging/tests/BUILD.gn

```diff
@@ -23,6 +23,8 @@ chip_test_suite("tests") {
 
   sources = [
     "TestExchangeMgr.cpp",
+    "TestHelpers.cpp",
```

Comment:
could we try to avoid names like 'helpers', 'utils' and 'common' as much as possible? It would help when writing new tests to have an idea of what these headers/cpp files contain.

I would propose 'MessageContext.h' or 'TestingIOContext.h' (if we end up renaming the class)

Reply:
Done